cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/ApacheKafkaTopicService.java

```diff
@@ -282,4 +331,24 @@ public synchronized ApiResponse deleteTopic(ClusterTopicRequest clusterTopicRequ
       throw e;
     }
   }
+
+  @Async("resetTopicsCacheTaskExecutor")
+  @Scheduled(
+      cron = "${klaw.topics.cron.expression:0 0 0 * * ?}",
+      zone = "${klaw.topics.cron.expression.timezone:UTC}")
+  public void resetTopicsCacheScheduler() {
+    topicCacheKeySets.forEach(
+        clusterKeyIdentifier -> {
+          try {
```

Comment:
If we added a timestamp for last load time, we could skip any reset on a cluster that has been recently started or completed?

Reply:
Having looked at the config the reset is less often then i thought so i think you can probably ignore this comment.

Reply:
Agree, will put into next pr on this.